### ISSUE TITLE
Values are not transmitted: {PATH}, {TARGET}, etc...

### ISSUE THREAD

**Issue by HideNsec** [yogeshojha/rengine#1010]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Values are not passed instead of custom tools

<img src="https://cdn.discordapp.com/attachments/880364122066272276/1165640632325386382/Screenshot_2023-10-22_at_16.18.32.png?ex=6547966f&is=6535216f&hm=29663cd50103a722e70447943eaa6272c4cb7d6903c231ff116e8603d14e57b0&" width="800" >

<img src="https://media.discordapp.net/attachments/880364122066272276/1165650796969480284/Screenshot_2023-10-22_at_17.00.02.png?ex=65479fe6&is=65352ae6&hm=1a7ac1717daf98b89bf6ee6d8905ea9a677fa6d09519ebbab178c24bd88b55d4&=&width=2160&height=1138" title="hover text">

<img src="https://media.discordapp.net/attachments/880364122066272276/1165651258615545896/Screenshot_2023-10-22_at_17.01.52.png?ex=6547a054&is=65352b54&hm=c069ce702d8eec3dab14b41ffcfa9b1a62fd21b8e80702a018d1c2f0e5deef0d&=&width=2000&height=1170" title="hover text">

<img width="974" alt="Screenshot 2023-10-22 at 23 00 51" src="https://github.com/yogeshojha/rengine/assets/148083631/227a7116-b13d-4d64-a5f6-fb2a32556e46">

The assumption that arose when trying to correct:

https://discord.com/channels/880363103689277461/880364122066272276/1165667120206590132

### Expected Behavior

Passing the host value to TARGET

### Steps To Reproduce

install rengine to ubuntu server 22.04 
add any custom tool 
make log
run scan via custom engine

### Environment

```markdown
- reNgine: 2.0.0
- OS: Ubuntu Server 22.04 LTS 
- Python: 3.10
- Docker Engine: -
- Docker Compose: -
- Browser: Safari, Google Chrome etc...
```

**Comment by yogeshojha** [comment]
@psyray can you please check this.

Thanks

**Comment by psyray** [comment]
@yogeshojha When I have time, yes ;)

**Comment by psyray** [comment]
@HideNsec According to the code you must supply **TARGET** and **OUTPUT**
https://github.com/yogeshojha/rengine/blob/fd5a5e5faa1e289cbf421e8aaf6014452efaef1f/web/reNgine/tasks.py#L470-L478

**Comment by psyray** [comment]
Fixed in #1125
